Package.swift

```diff
@@ -15,13 +15,13 @@ let package = Package(
     dependencies: [
         // dev .package(url: "https://github.com/danger/swift", from: "3.0.0"),
         // dev .package(path: "Submodules/WeTransfer-iOS-CI/Danger-Swift"),
-        // dev .package(url: "https://github.com/WeTransfer/Mocker.git", from: "2.1.0"),
+        .package(url: "https://github.com/WeTransfer/Mocker.git", from: "2.1.0"),
```

Comment:
These need to be put back so people that check out our package don't also checkout Mocker and our test target. This is fixed in Swift 5.2 but we did not upgrade to that yet.

```suggestion
        // dev .package(url: "https://github.com/WeTransfer/Mocker.git", from: "2.1.0"),
```